src/directives/NgGrid.ts

```diff
@@ -786,13 +792,16 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 			}
 			
 			if (shouldSave) {
-				collisions[0].savePosition(itemPos);
+				const newItemPos = this._fixGridPosition(itemPos, itemDims);
+				collisions[0].savePosition(newItemPos.col, newItemPos.row);
 			} else {
-				collisions[0].setGridPosition(itemPos);
+		                const newItemPos = this._fixGridPosition(itemPos, itemDims);

```

Comment:
I'm bumping it to 0.9.0 as it's a pretty hefty set of changes since the latest, and will push this evening